pkgs/top-level/all-packages.nix

```diff
@@ -17982,9 +17982,10 @@ with pkgs;
 
   dot2tex = with python3.pkgs; toPythonApplication dot2tex;
 
-  doxygen = callPackage ../development/tools/documentation/doxygen {
+
```

Comment:
```suggestion
```

Reply:
Adopted change locally; will be fixed next time I force-push.